frontend/src/react/Exercice/Exercice.js

```diff
@@ -17,14 +19,22 @@ export default class Exercice extends React.PureComponent {
 
   addQuestion = () => {
     const question = [...this.state.question];
-    question.push(1);
+    question.push({"questionTitle": '', "questionScale": null, "questionContent": ''});
+    this.setState({question});
+  };
+
+  delQuestion = (v) => {
+    console.log(v.target.value);
+    const question = [...this.state.question];
+    delete question[v.target.value];
     this.setState({question});
   };
 
   render() {
     return (
       <ExerciceDisplayer handleInput={this.handleInput}
                          addQuestion={this.addQuestion}
+                         delQuestion={this.delQuestion}
```

Comment:
Sorry I'm a fdp but could we use complete names like "deleteQuestion" ? 
(si je t'embête trop tu peux me le dire et on peut refactor plus tard)

Reply:
Not my work here, ask Felix or Arthur for more info

Reply:
@ArthurWirbel @FelixDelcourt :)

Reply:
Yeah sure dude, I'll change it asap, stay tuned 🕺

Reply:
I mean, change it before we merge please